homeassistant/components/litejet/light.py

```diff
@@ -91,16 +93,37 @@ def extra_state_attributes(self):
 
     def turn_on(self, **kwargs):
         """Turn on the light."""
+        is_complex = False
+        brightness = 99
+        transition = self._config_entry.options.get(CONF_DEFAULT_TRANSITION, 0)
+
         if ATTR_BRIGHTNESS in kwargs:
+            is_complex = True
             brightness = int(kwargs[ATTR_BRIGHTNESS] / 255 * 99)
-            self._lj.activate_load_at(self._index, brightness, 0)
+
+        if ATTR_TRANSITION in kwargs:
+            is_complex = True
+            transition = kwargs[ATTR_TRANSITION]
+
+        if is_complex:
+            self._lj.activate_load_at(self._index, brightness, int(transition))
         else:
             self._lj.activate_load(self._index)
 
     def turn_off(self, **kwargs):
         """Turn off the light."""
-        self._lj.deactivate_load(self._index)
+        is_complex = False
+        transition = self._config_entry.options.get(CONF_DEFAULT_TRANSITION, 0)
+
+        if ATTR_TRANSITION in kwargs:
+            is_complex = True
+            transition = kwargs[ATTR_TRANSITION]
+
+        if is_complex:
+            self._lj.activate_load_at(self._index, 0, transition)
+        else:
+            self._lj.deactivate_load(self._index)
```

Comment:
The default transition is never used.

```suggestion
        if ATTR_TRANSITION in kwargs:
            self._lj.activate_load_at(self._index, 0, kwargs[ATTR_TRANSITION])
            return

        self._lj.deactivate_load(self._index)
```